enterprise/server/cmd/ci_runner/main.go

```diff
@@ -151,6 +151,8 @@ var (
 	patchURIs          = flag.Slice("patch_uri", []string{}, "URIs of patches to apply to the repo after checkout. Can be specified multiple times to apply multiple patches.")
 	recordRunMetadata  = flag.Bool("record_run_metadata", false, "Instead of running a target, extract metadata about it and report it in the build event stream.")
 	gitCleanExclude    = flag.Slice("git_clean_exclude", []string{}, "Directories to exclude from `git clean` while setting up the repo.")
+	gitFetchFilters    = flag.Slice("git_fetch_filters", []string{}, "Filters to apply to `git fetch` commands.")
```

Comment:
Nit: I think we could get away with supporting a single filter currently thanks to https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---filterltfilter-specgt

Then perhaps if there is a valid use case, we could add support for multiple filters in the future.

Reply:
I considered this, but using multiple filters with the `combine` syntax requires %-encoding each of the individual filters which doesn't feel very user friendly to me. Using a slice here doesn't add much complexity so I think it's fine to start with a slice